<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>KST_EXPORT</exportmacro>
 <class>EventMonitorWidget</class>
 <widget class="QWidget" name="EventMonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>469</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxEMail">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>E-Mail Notification</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxEMailNotify">
        <property name="text">
         <string>N&amp;otify:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditEMailRecipients"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxEquation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Expression</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>Vectors:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Operators:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>comboBoxEqOperator</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxEqOperator">
        <item>
         <property name="text">
          <string>+</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>*</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>/</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>^</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PI</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>e</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&amp;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>|</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>!</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STEP()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ABS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SQRT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CBRT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SIN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TAN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ASIN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ACOS()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ATAN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SEC()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CSC()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COT()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SINH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COSH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TANH()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EXP()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LN()</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOG()</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ScalarSelector" name="_scalarSelectorEq" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel3">
        <property name="text">
         <string>Scalars:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditEquation"/>
      </item>
      <item row="1" column="1">
       <widget class="VectorSelector" name="_vectorSelectorEq" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="textLabel3_2">
        <property name="text">
         <string>Expression:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>lineEditEquation</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="textLabel3_2_3">
        <property name="text">
         <string>Description:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>lineEditDescription</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditDescription"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="groupBoxScript">
     <property name="title">
      <string>KstScript</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="_useScript">
        <property name="text">
         <string>Execute:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="_script"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBoxELOG">
     <property name="title">
      <string>ELOG</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxELOGNotify">
        <property name="text">
         <string>No&amp;tify:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_pushButtonELOGConfigure">
        <property name="text">
         <string>&amp;Configure...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>241</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Q3ButtonGroup" name="buttonGroupLog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Kst Debug Log</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="5">
       <spacer name="spacer1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>120</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioButtonLogWarning">
        <property name="text">
         <string>&amp;Warning</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="radioButtonLogError">
        <property name="text">
         <string>&amp;Error</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButtonLogNotice">
        <property name="text">
         <string>&amp;Notice</string>
        </property>
        <property name="buttonGroupId" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxDebug">
        <property name="text">
         <string>&amp;Log as:</string>
        </property>
        <attribute name="buttonGroup">
         <string/>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
  <customwidget>
   <class>ScalarSelector</class>
   <extends>QWidget</extends>
   <header>scalarselector.h</header>
  </customwidget>
  <customwidget>
   <class>VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxEqOperator</tabstop>
  <tabstop>_vectorSelectorEq</tabstop>
  <tabstop>_scalarSelectorEq</tabstop>
  <tabstop>lineEditEquation</tabstop>
  <tabstop>lineEditDescription</tabstop>
  <tabstop>checkBoxDebug</tabstop>
  <tabstop>radioButtonLogNotice</tabstop>
  <tabstop>radioButtonLogWarning</tabstop>
  <tabstop>radioButtonLogError</tabstop>
  <tabstop>checkBoxEMailNotify</tabstop>
  <tabstop>lineEditEMailRecipients</tabstop>
  <tabstop>checkBoxELOGNotify</tabstop>
  <tabstop>_pushButtonELOGConfigure</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
